source/features/deep-blame.tsx

```diff
@@ -44,13 +44,16 @@ const getPullRequestBlameCommit = mem(async (commit: string, prNumber: number):
 	return false;
 });
 
-function getCommitHash(commit: HTMLElement): string {
-	return (commit.nextElementSibling as HTMLAnchorElement).href.split('/').pop()!;
+function getCommitHash(commit: HTMLLinkElement): string {
+	return commit.href.split('/').pop()!;
 }
 
 async function redirectToBlameCommit(event: DelegateEvent<MouseEvent, HTMLAnchorElement>): Promise<void> {
 	const blameLink = event.delegateTarget;
-	const {lineNumber, commit: pullRequestCommit, prNumber} = blameLink.dataset;
+	const currentParentElement = blameLink.closest('.blame-hunk')!;
+	const lineNumber = select('.js-line-number', currentParentElement)!.textContent!;
+	const prNumber = looseParseInt(select('.issue-link', currentParentElement)!.textContent!);
+	const pullRequestCommit = getCommitHash(select<HTMLLinkElement>('a.message', currentParentElement)!);
```

Comment:
It's used once, you can inline it
```suggestion
	const pullRequestCommit = select<HTMLAnchorElement>('a.message', currentParentElement)!.href.split('/').pop()!;
```

Reply:
>HTMLLinkElement is <link> 
oh. What is an A?